main.go

```diff
@@ -1,82 +1,74 @@
 package main
 
 import (
-	"github.com/hashicorp/go-getter"
-	"gopkg.in/go-playground/webhooks.v5/github"
-	"io"
-	"log"
 	"net/http"
 	"os"
-)
-
-const (
-	path = "/payload"
-	port = ":3069"
-)
 
-var (
-	Info    *log.Logger
-	Warning *log.Logger
-	Error   *log.Logger
+	"github.com/joho/godotenv"
 )
 
-func setLogStreams(
-	infoHandle io.Writer,
-	warningHandle io.Writer,
-	errorHandle io.Writer) {
-
-	Info = log.New(infoHandle,
-		"INFO: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
-	Warning = log.New(infoHandle,
-		"WARNING: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
-	Error = log.New(infoHandle,
-		"ERROR: ",
-		log.Ldate|log.Ltime|log.Lshortfile)
+func init() {
+	// Setup log streams
+	setLogStreams(os.Stdout, os.Stdout, os.Stderr)
 
+	//Load .env file
+	if err := godotenv.Load(); err != nil {
+		Error.Print("No .env file found")
+	}
```

Comment:
You don't want to do this. This prevents the programmer for overriding the `.env` file. Instead they need to source it before running you